### ISSUE TITLE
Request for clarification on layer height

### ISSUE THREAD

**Issue by tumes** [riskable/void_switch#9]
I've actually printed a fair few of these and I'm loving them, but I have a question regarding the layer height recommendation since I've been post-processing them and wanted to make sure I was being maximally efficient. When you recommend 0.16 for layer height, does that also include the first layer? I'm asking because the slicer I use (Prusa) defines the first layer height separately from the subsequent ones, and depending on which profile you're starting from I reckon that disparity could account for all sorts of quirkiness.

Regardless thanks for your hard work! I fabbed up some of your beta PCBs and I'm looking forward to slowly building this out!

**Comment by riskable** [comment]
Yeah, the 0.16mm layer height also includes the first layer.  It doesn't matter *that* much though because all that happens if you choose an indivisible layer height is you get slightly more wobble on the top-left/bottom-right range of motion.  It really doesn't matter that much unless you *really* hate even the tiniest wobble in your switches (which only happens when your fingers are on them--because the magnets hold them steady when at rest regardless).
